_docs/concepts/auth.md

```diff
@@ -0,0 +1,138 @@
+---
+category: Concepts
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
+parent: Network and Auth
+order: 20
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+The page explains Auth's role and general architecture.
+
+
+## Istio Auth Concept
+
+*Status: Draft*
+
+*Date: April, 2017*
+
+
+[TOC]
+
+
+### Overview {#overview}
+
+Istio Auth aims at enhancing the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Presenting a strong identity that represents the role of the service to enable interoperability across clusters and clouds.
```

Comment:
"Presenting" is a bit odd here. "Providing each service with a strong identity that represents its role to enable..."

(if that doesn't change the meaning of the sentence...)